charters/charter-2021.html

```diff
@@ -208,21 +208,21 @@ <h2>Success Criteria</h2>
         <ul>
           <li>Identification of requirements for tighter support of
             media-centric applications on the Web platform, possibly through
-            the creation of Task Forces</li>
+            the creation of Task Forces.</li>
           <li>Incubation of technical solutions for the identified requirements by
             supporting the creation of or discussion in existing Community Groups or Business Groups focusing on
-            particular topics
+            particular topics.
           </li>
-          <li>Suggesting existing Working Groups to include particular topics in
+          <li>Suggesting existing Working Groups that they include particular topics in
```

Comment:
I think it needs a "to":

```suggestion
          <li>Suggesting to existing Working Groups that they include particular topics in
```